docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,353 @@
+```
+  </TabItem>
+</Tabs>
+
+#### Dictionary
+
+You can specify options as a dictionary when initializing the primitive.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+ ```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"resilience_level": 2, "resilience": {"zne_mitigation": True, "zne": {"noise_factors": [1, 3, 5]}}})
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+ ```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"resilience_level": 2})
+```
+  </TabItem>
+</Tabs>
+
+### Update options after initialization
+
+You can specify the options in this format: `primitive.options.option.sub-option.sub-sub-option = choice` to take advantage of auto complete, or use the `update()` method to make bulk updates.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+estimator = Estimator(mode=backend)
+
+# Setting options after primitive initialization
+# This uses auto complete.
+estimator.options.default_shots = 4000
+# This does bulk update.
+estimator.options.update(default_shots=4000, resilience={"zne_mitigation": True})
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+```python
+# Setting options after primitive initialization
+# This uses auto complete.
+options.resilience_level = 2
+estimator = Estimator(backend=backend, options=options)
+
+# Setting options after primitive initialization.
+# This does bulk update.
+estimator.set_options(shots=4000)
+```
+  </TabItem>
+</Tabs>
+
+<span id="run-method"></span>
+### Run() method
+
+The run() method accepts options in V1 only.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+In V2, the only values you can pass to `run()` are those defined in the interface.  That is, `shots` for Sampler and `precision` for Estimator. This overwrites any value set for `default_shots` or `default_precision` for the current run.
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+sampler = Sampler(mode=backend)
+# Sample two circuits at 128 shots each.
+sampler.run([circuit1, circuit2], shots=128)
+
```

Comment:
Adding `default_shots` to the code example since it's mentioned in the description. 

```suggestion
sampler = Sampler(mode=backend)

# Default shots to use if not specified in run()
sampler.options.default_shots = 500

# Sample two circuits at 128 shots each.
sampler.run([circuit1, circuit2], shots=128)

```